### PR TITLE
publishing velocity and position to control state in ekf2

### PR DESCRIPTION
**Pull request by CarlOlsson**
This needs to be tested in SITL before merged

### CODE REVIEW

File: src/modules/ekf2/ekf2_main.cpp [hunk 1]
```diff
@@ -484,6 +484,25 @@ void Ekf2::task_main()
 			ctrl_state.pitch_rate = _lp_pitch_rate.apply(sensors.gyro_rad_s[1]);
 			ctrl_state.yaw_rate = _lp_yaw_rate.apply(sensors.gyro_rad_s[2]);
 
+			/* Velocity in Body Frame */
+			float velocity[3];
+			_ekf->copy_velocity(velocity);
+			Vector3f v_n(velocity);
+			matrix::Dcm<float> Tnb(q);
+			Vector3f v_b = Tnb * v_n;
```
Comment (tumbili) [inline src/modules/ekf2/ekf2_main.cpp]: @sverling Is this inteded to be in body frame? Just asking because the position below is in earth frame.

Comment (sverling) [inline src/modules/ekf2/ekf2_main.cpp]: the idea was to implement it the same way it is done in [ekf_att_pos_estimator](https://github.com/PX4/Firmware/blob/43e0186eb383bbbbd13fb9afbec95530d0fb643f/src/modules/ekf_att_pos_estimator/ekf_att_pos_estimator_main.cpp#L878:L880)

**Comment by tumbili** [comment]
@sverling Ok, cool. So could you give this a try in SITL. I'm not sure if we are using the control state velocities anywhere in the code so probably nothing will change with this. However, we should still check?

**Comment by tumbili** [comment]
Testing this today in combination with ekf2 airspeed fusion.

**Pull request closed without merging**